developer_manual/design/layout.rst

```diff
@@ -11,14 +11,15 @@ While deciding how you want your app to look, there are a number of factors to c
 * Typical interface patterns in other similar apps in the market
 
 The `content Vue component <https://nextcloud-vue-components.netlify.app/#/Components/App%20containers?id=ncappcontent>`_ wraps your entire app. While the arrangement of components in your app is dependent on what your app does, most Nextcloud apps typically have 3 levels of hierarchy. Some commonly used layouts are:
+`Penpot empty template https://design.penpot.app/#/view/db3839da-807b-8052-8002-576401e9a375?page-id=2783d7ad-98f2-804a-8002-750c2585d4f1&section=interactions&index=5&share-id=11fde340-21f4-802e-8002-8d8d305e7ab5`_
```

Comment:
Same here, best to use the same unified Penpot link as in https://github.com/nextcloud/documentation/pull/10431 :)

Reply:
This is the same link like in #10431 :)

Reply:
true. not sure how that happened :D